content/resources/powershell_script/_index.md

```diff
@@ -13,44 +13,38 @@ menu:
     identifier: chef_infra/cookbook_reference/resources/powershell_script powershell_script
     parent: chef_infra/cookbook_reference/resources
 resource_description_list:
-- markdown: 'Use the **powershell_script** resource to execute a script using the
+- markdown: |-
+    Use the **powershell_script** resource to execute a script using the Windows PowerShell interpreter, much like how the script and script-based resources—**bash**, **csh**, **perl**, **python**, and **ruby**—are used. The powershell_script is specific to the Microsoft Windows platform and the Windows PowerShell interpreter.
 
-    Windows PowerShell interpreter, much like how the **script** and
-
-    **script**-based resources---**bash**, **csh**, **perl**, **python**,
-
-    and **ruby**---are used. The **powershell_script** is specific to the
-
-    Microsoft Windows platform and the Windows PowerShell interpreter.
-
-
-    The **powershell_script** resource creates and executes a temporary
-
-    file (similar to how the **script** resource behaves), rather than
-
-    running the command inline. Commands that are executed with this
-
-    resource are (by their nature) not idempotent, as they are typically
-
-    unique to the environment in which they are run. Use `not_if` and
-
-    `only_if` to guard this resource for idempotence.'
+     The powershell_script resource creates and executes a temporary file (similar to how the script resource behaves), rather than running the command inline. Commands that are executed with this resource are (by their nature) not idempotent, as they are typically unique to the environment in which they are run. Use `not_if` and `only_if` conditionals to guard this resource for idempotence.
```

Comment:
```suggestion
     The powershell_script resource creates and executes a temporary file rather than running the command inline. Commands that are executed with this resource are (by their nature) not idempotent, as they are typically unique to the environment in which they are run. Use `not_if` and `only_if` conditionals to guard this resource for idempotence.
```

The `script` resource doesn't use a temp file anymore.

Reply:
I can open a PR to fix this on the resource.